manual/cli.rst

```diff
@@ -1,4 +1,100 @@
++-----------+----------------------------+------------------------------------------------------------------------------------------------------+
+| -f N      | --resize-factor=N          | resize factor                                                                                        |
++-----------+----------------------------+------------------------------------------------------------------------------------------------------+
+|           | --sea_color                | help="string for color [blue|brown], [default = brown]                                               |
++-----------+----------------------------+------------------------------------------------------------------------------------------------------+
+|           | --not-draw-biome           | Not draw biome                                                                                       |
++-----------+----------------------------+------------------------------------------------------------------------------------------------------+
+|           | --not-draw-mountains       | Not draw mountains                                                                                   |
++-----------+----------------------------+------------------------------------------------------------------------------------------------------+
+|           | --not-draw-rivers          | Not draw rivers                                                                                      |
++-----------+----------------------------+------------------------------------------------------------------------------------------------------+
+|           | --draw-outer-border        | Draw outer land border                                                                               |
++-----------+----------------------------+------------------------------------------------------------------------------------------------------+
+
+Export options
+~~~~~~~~~~~~~~
+
++-----------+----------------------------+--------------------------------------------------------------------------------------------------------------------------------+
+| Short     | Long                       | Description                                                                                                                    |
++===========+============================+================================================================================================================================+
+|           | --export-format            | Export to a specific format such as BMP or PNG. See http://www.gdal.org/formats_list.html for possible formats [default = PNG] |
++-----------+----------------------------+--------------------------------------------------------------------------------------------------------------------------------+
+|           | --export-datatype          | Type of stored data, e.g. uint16, int32, float32 etc. [default = uint16]                                                       |
```

Comment:
These two are my own mistakes: These parameters need to be like `--export-format='PNG'` etc. I obviously made a mistake, and now it looks as if the user wouldn't have to pass in anything.
I will try to update this - this part of the manual is probably not generated automatically, is it?